doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+      \vdots & \vdots & \ddots & \vdots \\
+      P_{s_2,n_2,ay_1 \rightarrow s_1,n_j,ay_2} & P_{s_2,n_2,ay_1 \rightarrow s_2,n_n,ay_2} & \dots & P_{s_2,n_2,ay_1 \rightarrow s_i,n_j,ay_2} \\
+    };
+
+    \draw[dotted](mC-2-2)--(mD.north);
+  \end{tikzpicture}
+  \caption{Markov Transition Matrix (many nested probability
+    matrices)\label{transition matrix}
+  }
+\end{figure}
+
+In the Fig.\ref{transition matrix} every $P_{s_i,n_j,ak_1}$ is a
+probability matrix that represents the probabilities of transitioning
+to any other state in the next academic year (see the diagram's
+cut-out for an example.)
+
+Armed with $P$, this rather complicated Markov Transition Matrix, we can
+apply it to the initial state
+
+$Y_1 = Y_0 \cdot P$
+
+In fact at this point as $P$ is time independent we can work the
+populations in year $n$ by
+
+$Y_n = Y_0 \cdot P^n$
+
+Of note is that this equation requires no tracking of $t$'s we can
+simply use the above formula to work out $Y_n$. BUT we don't do this!
+Why?
+
+\subsubsection{Send is not MC}
+
+The send model actually sides step the creation of a Markov Transition
+Matrix and takes an algorithmic approach to calculating $Y_{n+1}$'s
+population.
+
+This is done as some of the data is perhaps too sparse, but also the
+algorithmic approach allows us to introduce priors and scenarios
+rather more easily than modifying $P$ directly.
+
+Further, we also use more than the just the initial state, in our
+algorithmic predictions.  The model incorporates external population
+data for each predicted calendar year.
+
+As we will come to see the algorithmic approach to working out the
+$t$'s in $Y_n$ introduces stochastic processes. Thus we introduce the
+need to use Monte Carlo methods to ensure confidence in our
+calculations.
+
+The model can be thought of as a simulation in with a simulation clock
+that ticks once a calendar year. On clock tick all the entities have
+their new population calculated: based on their existing state,
+pre-calculated historic probability distributions and projected
+population.
+
+\subsection{Send Model's Population Prediction Algorithm}
+
+The algorithm can be simplified to:
+
+$Y_{n+1} = T(Y_n)$
+
+$Y_n$ represents calendar year n's population state, $T$ the function
+that calculates the next years population state from it.  The
+simplification of $T$ means that some things that are variable are
+closed over i.e, the external ONS population data is not a variable,
+nor are the pre-calculated probability distributions.
+
+Let us consider first a single entity's population $t_{s_x,n_y,ay_z}$.
+How will $T$ operate on this?
+
+\subsubsection{Movers,  Non-Movers, Leavers and Joiners}
+
+The model introduces the concepts of movers($m$), leavers($l$), and
+joiners($j$).
+
+Let $m_{s_x,n_y,ay_z}$ be defined as all the pupils that move
+\textbf{into} a particular entity from another entity.
+
+Let $n_{s_x,n_y,ay_z}$ be defined as all the pupils that
```

Comment:
I would imagine that, as defined here, nobody is a "non-mover" since they always progress 1 academic year even if nothing else changes?